app/controllers/places_controller.rb

```diff
@@ -8,6 +8,9 @@ def index
 
   # GET /places/1 or /places/1.json
   def show
+    @place = Place.find(params[:id])
+    @storage_item = @place.storage_item
```

Comment:
There will be multiple storage items for each place, so this would be more like `@storage_items = @place.storage_items"